src/layouts/Login/components/LoginForm.tsx

```diff
@@ -0,0 +1,64 @@
+import { FormControl } from "@chakra-ui/react"
+import {
+  Button,
+  FormLabel,
+  Input,
+  FormErrorMessage,
+} from "@opengovsg/design-system-react"
+import { useEffect } from "react"
+import { useForm } from "react-hook-form"
+
+export type LoginProps = {
+  email: string
+}
+
+interface LoginFormProps {
+  onSubmit: (inputs: LoginProps) => Promise<void>
+  errorMessage: string
+}
+
+const validateEmail = (value: string) =>
+  value.length > 0 || "Please enter a valid email."
```

Comment:
out of curiosity, how does work with `<Input type = "email" />`? will it only allow valid emails or will there also be invalid inputs?

Reply:
You can still get some invalid inputs (e.g. `a@a` is accepted), but it's probably better to include anyway

Reply:
ah cool thanks 4 explaining